engine/examples/dotnetcore.md

```diff
@@ -0,0 +1,102 @@
+---
+description: Create a Docker image by layering your ASP.NET Core app on debian
+for Linux Containers or with Windows Nano Server containers using a Dockerfile.
+keywords: docker, dockerize, dockerizing, dotnet, .NET, Core, article, example,
+platform, installation, containers, images, image, dockerfile, build, asp.net,
+asp.net core title: Dockerizing a .NET application
+---
+
+## Introduction
+
+This example demonstrates how to dockerize an ASP.NET Core application.
+
+## Why build ASP.NET Core?
```

Comment:
Can you add a blank line after all headings? It's consistent with other documents; also, I seem to recall some Markdown renderers having issues if the blank line is missing (but maybe that's no longer the case)